kpconv_torch/utils/config.py

```diff
@@ -385,3 +399,29 @@ def save(self):
                 text_file.write(f"epoch_steps = {self.epoch_steps:d}\n")
             text_file.write(f"validation_size = {self.validation_size:d}\n")
             text_file.write(f"checkpoint_gap = {self.checkpoint_gap:d}\n")
+
+    def set_infered_file_path(self, path):
+        self.infered_file = path
+    
+    def get_infered_file_path(self):
+        return self.infered_file
+
+    def get_test_save_path(self):
```

Comment:
Retourne le bon chemin où sauvegarder la simulation, selon les options utilisées dans la CLI

Reply:
If there is a `infered_file`, the output is stored near to it. If not, we store the output in the log folder.

Why don't we store the output in the log folder systematically? (In this case, no need for extra-`if` nor `self.infered_file`.)

Reply:
I find it more logical to find the output near the infered file and not in another place.

Reply:
My point: the output could have been stored in the training session directory (=the log directory): in this scenario, every inference process result is stored near to the model which has been used to produce it.

But fine, let's assume writing the inference output near to the infered file is a convenient solution too, that's sounds OK to me.